apps/generator/docs/configuration-file.md

```diff
@@ -55,6 +57,15 @@ The `generator` property from `package.json` file must contain a JSON object tha
         }
     }
   },
+  "conditionalGeneration": {
+     "directoryName/fileName": { 
+       // Name of the directory or fileName without ./ or any kind of path, e.g., "docs" or "output"
+     "parameter": "singleFile", // The template parameter that will determine whether the directory is generated
+     "validation": {
+       // The validation rule that determines when the directory should be generated
+       "not": { "const": true } // This rule ensures the directory is not generated if "singleFile" is true
+    }
+  },
```

Comment:
github will always confuse us and highlight this as error as comments are not allowed in json

basically modify it in the way that there are 2 examples, one for directory and one for file, and simply put names that explain what feature shows, like for example `conditionOnFile.js` or `consition/on/directory`

you anyway need to show also a example that is shown above with `subject` feature

Reply:
also remove entirely below, no need to promote feature that is deprecated

```
"conditionalFiles": {
    "path/to/file/that/is/relative/to/template/dir/test-amqp.js": {
      "subject": "server.protocol",
      "validation": {
        "const": "amqp"
      }
    },
```

Reply:
Done thanks @derberg